### ISSUE TITLE
Horizontal buttons stack > Highlight Current View is not activating

### ISSUE THREAD

**Issue by Micah-S** [Clooos/Bubble-Card#569]
The `highlight_current_view` option for a Horizontal buttons stack is not applying the appropriate classname when on the selected page.

I did some digging and noticed the event listener is never firing due to being an invalid event type:
https://github.com/Clooos/Bubble-Card/blob/37be1b9bcbf1b856b8b2f9cb8266632b718d5de4/src/cards/horizontal-buttons-stack/create.ts#L65

The correct method is documented [here](https://stackoverflow.com/a/52809105):
```
window.navigation.addEventListener('navigate', handleUrlChange); 
```

**Comment by Clooos** [comment]
Nice catch! It was indeed something we forgot to change during the refactoring.

I will fix that in the next release!

**Comment by Clooos** [comment]
This is now fixed in v2.0.4, thanks again for the feedback!

https://github.com/Clooos/Bubble-Card/releases/tag/v2.0.4
